lab_2_tokenize_by_bpe/main.py

```diff
@@ -14,6 +14,18 @@ def prepare_word(
     :param end_of_word: a token that signifies the end of word
     :return: preprocessed word
     """
+    if not isinstance(raw_word, str) or (start_of_word is not None and
+                                         not isinstance(start_of_word, str)) or (
+            end_of_word is not None and not isinstance(end_of_word, str)):
+        return None
+    total_result = []
+    if start_of_word and start_of_word != "":
```

Comment:
Yeah well you did fix it! Nice thinking.

However, I was referring to the fact that `None` is a singleton, and better comparison with it would be with `is` operator

Reply:
for EoW the same thing

Reply:
I've read some more information and hope that I got you this time, I'm sorry

Reply:
Please use `is`